examples/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/assets/case_8.py

```diff
@@ -0,0 +1,33 @@
+from dagster import OpExecutionContext, ResourceParam
+from dagster_dbt.asset_decorators import dbt_multi_asset
+from dagster_dbt.cli.resources_v2 import DbtClientV2, DbtManifest
+
+from tutorial_dbt_dagster_v2.assets import manifest
+
+
+## Case 8: run subsets of the dbt graph.
+@dbt_multi_asset(manifest=manifest)
+def dbt_assets(context: OpExecutionContext, dbt: ResourceParam[DbtClientV2]):
+    dbt_subset_selection = DbtManifest(manifest=manifest).get_subset_selection_for_context(
+        context=context
+    )
+    dbt_commands = (
+        [
+            ["seed", "--select", dbt_subset_selection],
+            ["run", "--select", dbt_subset_selection],
+            ["test", "--select", dbt_subset_selection],
+        ]
+        if dbt_subset_selection
```

Comment:
I think we could just have the user use `dbt_subset_selection` in all cases, which would simplify this a bit. Maybe just calling it `dbt_selection_for_context` would be clearer in that case. An added bonus of that approach is that if we wire the raw select string from the decorator into the op metadata / tags, then we avoid making the use duplicate their raw select string from the decorator into the body of the asset